.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
is there a common prettier config that is used across FT apps? similar to the `eslint` config?

Reply:
This is copied from Page Kit, I'm not aware of other projects using prettier but if there are we should consider publishing a common config like we do with eslint.

Probably out of scope for this given what we know atm.

Reply:
It looks like a lot of changes comes from removing the semi-colon at the end of the line. Should we keep it?

Reply:
I personally prefer to keep the configuration to stock and not change it as style is subjective.

Semi-colons or no colons is controversial so the intention here is to mirror Page Kit, so I'd say any discussion about our general style should be had when/if we consider a single prettier config for FT.